src/runtime/hexagon_remote/bin/src/halide_hexagon_remote_stub.c

```diff
@@ -20,216 +20,215 @@
 #ifdef _WIN32
```

Comment:
Is this machine-generated? If so, we should exclude it

(maybe we should exclude all the files in this dir anyway?)